### ISSUE TITLE
Inaccurate results

### ISSUE THREAD

**Issue by krusej23** [andrewseago/Public-Scripts#1]
I ran the script and it outputted everything fine. I am running through the results now while looking at Casper Admin. It is reporting that some of my .dmgs are not being used but I know they are. I click on that same dmg in Casper Admin and it shows that it's being used in one of our configurations. So I'm just double checking everything but this script is awesome for cleaning up pkgs, dmgs, and scripts. Thank you!
![screen shot 2015-01-15 at 11 42 07 am](https://cloud.githubusercontent.com/assets/10549869/5762941/98421fdc-9cab-11e4-9d0d-bb281eb3c068.png)

**Comment by andrewseago** [comment]
I agree. I was seeing that myself and just haven't had time to revisit it yet.
